.travis.yml

```diff
@@ -8,16 +8,13 @@ addons:
   google-chrome: latest
 
 before_script:
-  - gulp
-  - bundlesize -f dist/sweetalert2.all.min.js -s 15kB
+  - npm run build-dist
 
 script:
-  # unit tests against dist/sweetalert2.js
-  - xvfb-run testem ci --reporter dot
-
-  # unit tests against dist/sweetalert2.all.min.js
```

Comment:
Are we dropping tests against `all.min.js`? Why?

Reply:
I removed this bit when I was struggling with the CI environment...
https://github.com/sweetalert2/sweetalert2/pull/826/commits/b2a23ca4995c3f3e7ef169d234d32d4aed8bc4f4

Embarrassingly, I can not remember why, but.. It seems redundant to run *all* the same tests (of business logic) against this different version, when we just want to check that the build infrastructure works properly.

Perhaps it would be better if we run the tests only once, but against the `all.js` build (rather than the plain `js` build)?

Either way, since this runs on CI, I am ok to keep the tests against `all.min.js` if you prefer.

Reply:
There was a case when `.js`+`.css` were working fine, but `all.js` was broken: https://github.com/sweetalert2/sweetalert2/issues/673#issuecomment-339105796

In order to avoid such weird issues in future, there are tests against both `.js`+`.css` and `all.js`.